ms/cv/cv.less

```diff
@@ -9,6 +9,7 @@
             padding: 0px 30px 0px 17px;
         }
 
-        h5 { padding: 0;margin: 0;}
+        h5 { padding: 0;margin: 0;
+        font-family: "HelveticaBold";}

```

Comment:
formatting, it should look like :
h5: {
   padding: 0;
   margin: 0;
   font-family: ...
}